src/components/keys.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-unused-vars */
+import { operator } from "./utils";
+const Keys = ({
+  keys,
+  getNumber,
+}: {
+  keys: { num: string }[];
+  getNumber: (numkey: string) => void;
```

Comment:
Consider changing the name of the function or its return type cause it is kind of confusing the way getNumber returns void
But it's optional if there is a reason behind